pontos/changelog/main.py

```diff
@@ -18,16 +18,17 @@
 import sys
 from argparse import ArgumentParser
 from pathlib import Path
-from typing import Iterable, NoReturn
+from typing import NoReturn, Optional, Sequence
 
 from pontos.changelog.conventional_commits import ChangelogBuilder
 from pontos.errors import PontosError
 from pontos.terminal.null import NullTerminal
 from pontos.terminal.rich import RichTerminal
+from pontos.terminal.terminal import Terminal
 from pontos.version.helper import get_last_release_version
 
 
-def parse_args(args: Iterable[str] = None) -> ArgumentParser:
+def parse_args(args: Optional[Sequence[str]] = None) -> ArgumentParser:
```

Comment:
From looking at the function the return type seems to be wrong to me. Should be `Namespace` from `argparse`.
```suggestion
def parse_args(args: Optional[Sequence[str]] = None) -> Namespace:
```

Reply:
Thanks, saved a lot of `#type: ignore` now :D